src/index.js

```diff
@@ -4,6 +4,9 @@ import App from './App';
 import * as serviceWorker from './serviceWorker';
 import './semantic/dist/semantic.min.css';
 import { BrowserRouter } from 'react-router-dom'
+import axios from 'axios'
+
+axios.defaults.baseURL = 'http://localhost:3002'
 
```

Comment:
try to make 3 letter variable for that. so that we don't have to copy paste the long string every time. also cause i'm curious if a variable would work there :P

Reply:
tried it on another PR. it works.